### PR TITLE
Set grad to False in VGG19 correctly 

### PR DESCRIPTION
**Pull request by BrunoEduardoCSantos**
Hi,
I have just corrected the way to set false the grad of vgg19 parameters like:
`param.required_grad= False`
Cheers,
Bruno

### CODE REVIEW

**Comment by iamRusty** [comment]
Hello @BrunoEduardoCSantos,
There's no problem with `param.requires_grad_(True)`, it works perfectly! 
Your suggested solution, however, doesn't work. If you really want to explicitly assign a value for the gradient tracking, you should use `param.requires_grad = False`, instead of `param.required_grad = False`. Notice the difference between requireS and requireD. 

Nonetheless, I believe that this change in the approach of turning off gradient tracking, is not needed for the notebook. :)

**Comment by BrunoEduardoCSantos** [comment]
Thanks for your feedback. :) It makes sense.
